huaweicloud/services/dms/resource_huaweicloud_dms_kafka_instance.go

```diff
@@ -288,6 +290,10 @@ func ResourceDmsKafkaInstance() *schema.Resource {
 					},
 				},
 			},
+			"charging_mode": common.SchemaChargingMode([]string{}),
+			"period_unit":   common.SchemaPeriodUnit([]string{}),
+			"period":        common.SchemaPeriod([]string{}),
+			"auto_renew":    common.SchemaAutoRenewUpdatable(nil),
```

Comment:
`auto_new` can be updated, but it's not used in Update method.

Reply:
this param can not be updated in this resource, so it has been marked as ForceNew